app/templates/admin/createEvents.html

```diff
@@ -19,6 +19,20 @@
     <h1 id ="title">{{program.programName}}</h1>
     <h2> New Event Form</h2>
   </div>
+
+  {% if 'edit_event' in request.path and g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentStaff %}

```

Comment:
Use parentheses to avoid thinking about order of operations - I think this won't evaluate how you expect.